spec/sidekiq_publisher/report_unpublished_count_spec.rb

```diff
@@ -32,5 +32,17 @@
       expect(instrumenter).to have_received(:instrument).
         with("unpublished.reporter", unpublished_count: job_count)
     end
+
+    context "and a PG::ConnectionBad/PG::UnableToSend error is encountered" do
+      before do
+        allow(SidekiqPublisher::Job).to receive(:unpublished).and_raise(PG::ConnectionBad)
```

Comment:
Quick note here, I originally had a [spec](https://github.com/ezcater/sidekiq_publisher/blob/43d9c6ae72538eaf484c7c9b88930b9e2e38fe02/spec/sidekiq_publisher/report_unpublished_count_spec.rb#L36-L55) that would actually disconnect the Postgres connection but this was causing [issues in the rails 5.1 spec](https://app.circleci.com/jobs/github/ezcater/sidekiq_publisher/150). Unfortunately, that disconnection cause other specs to fail so ended up going with this route